src/applications/personalization/connected-accounts/containers/ConnectedAcctApp.jsx

```diff
@@ -51,16 +47,13 @@ class ConnectedAcctApp extends React.Component {
         <ConnectedApps
           confirmDelete={this.confirmDelete}
           accounts={this.props.accounts}
-          propertyName={propertyName}
+          propertyName={'VA.gov'}
```

Comment:
Out of scope for this, but maybe we could refactor `ConnectedApps` and `NoConnectedApps` so they don't need the `propertyName` passed into them :-D

Reply:
Nitpicky, but you could also switch to `propertyName="VA.gov"` here and below.

Reply:
I went ahead and removed that `propertyName` prop. I'm not sure when we'll get around to it otherwise.